### PR TITLE
Made loose pipes dangerous

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Loose atmospherics pipes are now dangerous to be hit by.
/:cl:

Makes them embeddable, just like metal rods, and gives them throw_force.
This makes the singularity just a tiny bit more threatening, with all that
debris flying everywhere.

### CODE REVIEW

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -801,8 +801,9 @@ var/global/list/common_tools = list(
 	if(is_pointed(W))
 		return 1
 
-	var/list/embed_items = list(\
-	/obj/item/stack/rods,\
+	var/list/embed_items = list(
```
Comment (RemieRichards) [inline code/__HELPERS/unsorted.dm]: bonus points:
Update these procs' lists to be typecaches.

**Comment by RemieRichards** [comment]
I did say "these procs" not "this proc" so "TWO items"'s passive aggressiveness isn't really waranted.

**Comment by WJohn** [comment]
So now you can use these in a pressure cannon? Not like it's much of a buff, glass shards and rods can already be stuffed until they can one-shot somebody.

**Comment by FantasticFwoosh** [comment]
Time to throw a sorium grenade between a full stack of pipes for a shrapnel bomb.

**Comment by ExcessiveUseOfCobblestone** [comment]
*spams pipes*

I WAS JUST MAKING A SUPER LETHAL KILLING DEVICE MR ADMIN

**Pull request merged**